desktop-app/app/components/NetworkProxy/ProxyManager/styles.css

```diff
@@ -0,0 +1,45 @@
+.proxyManagerContainer {
```

Comment:
To center on a styling technology all styles should be applied using material-ui.

Reply:
please provide an example of how can I achieve this, I simply followed other components structure

Reply:
Sure. In the js file you can use `makeStyles`.

```js
import {makeStyles} from '@material-ui/core/styles';
// ...
function Component() {
  const classes = useStyles();
  // ...
  return (
    <div className={classes.bookmarks}>
      // ...
    </div>
  )
}
// ...
const useStyles = makeStyles(theme => ({
  bookmarks: {
    padding: '0 10px',
    display: 'flex',
    marginTop: '4px',
  },
}));
```

https://material-ui.com/styles/basics/

In case you were dealing with a class component you can use the `withStyles` HOC.

Reply:
done @jzabala